components/channel_header_dropdown/channel_header_dropdown_items.tsx

```diff
@@ -29,6 +28,11 @@ import ChannelPermissionGate from 'components/permissions_gates/channel_permissi
 
 import Menu from 'components/widgets/menu/menu';
 
+import {PluginComponent} from 'types/store/plugins';
+import {UserProfile} from '@mattermost/types/users';
+
+import {Channel} from '@mattermost/types/channels';
```

Comment:
```suggestion
import {PluginComponent} from 'types/store/plugins';

import {UserProfile} from '@mattermost/types/users';
import {Channel} from '@mattermost/types/channels';
```

Reply:
https://github.com/mattermost/mattermost-webapp/pull/11195/files#r984750473 i mean like this. we try to keep imports from same packages combined for easy viewing